Preserve distributionSha256Sum in gradle-wrapper.properties by Wrappe… (#22220)
gradle/gradle: subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

=== Potential breaking changes

=== Wrapper task configuration

> If `gradle-wrapper.properties` contains `distributionSha256Sum` property, then you also have to specify one in the wrapped task configuration or with the `--gradle-distribution-sha256-sum` task option.

Reviewer: ```suggestion
If `gradle-wrapper.properties` contains the `distributionSha256Sum` property, you must specify a sum. You can specify a sum in the wrapped task configuration or with the `--gradle-distribution-sha256-sum` task option.
```
Small wording improvement + breaking up a large multi-thought sentence.
